src/Sulu/Bundle/CategoryBundle/Resources/public/js/components/categories/list/main.js

```diff
@@ -37,7 +37,16 @@ define(function () {
                 toolbar: {
                     buttons: {
                         add: {},
-                        deleteSelected: {}
+                        deleteSelected: {},
+                        export: {
+                            options: {
+                                urlParameter: {
+                                    flat: true,
+                                    limit: 999999
```

Comment:
Shouldn't it be possible to omit this, and really export everything? That's the actual intention of the export, isn't it?